### DISCUSSION TITLE
Can't start CSGO server, it says: executable was not found srcds_run

### DISCUSSION THREAD

**Discussion by gbiatturi** [GameServerManagers/LinuxGSM#3426] [Q&A]
Hi! When I use './csgoserver start' it says:

**Starting csgoserver: executable was not found
* /home/csgoserver/serverfiles/srcds_run**

I checked for updates doing './csgoserver cu' and tried validating SteamCMD with './csgoserver v' but it seems ok.

Any ideas? Thanks!

**Comment by neerro** [comment]
Same issue...
